lib/analyser.js

```diff
@@ -126,19 +128,23 @@ module.exports = {
    * @param {array} whitelist
    */
   logResults(showStats, showOccurrences, whitelist) {
-    let percentage = (this.unusedComponents.length / this.components.length) * 100;
+    let unusedComponentCount = Object.keys(this.unusedComponents).length;
+    let componentCount = Object.keys(this.components).length;
```

Comment:
Use `componentCount` in line `167`, otherwise I see `NaN%`:

![image](https://user-images.githubusercontent.com/4335166/68391196-50f95c00-0167-11ea-8d20-95997faa6147.png)